mobile/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: get_10101
 description: 10101 combines the power of a self-custodial on-chain and off-chain wallet with the vast world of trading.
 publish_to: none
-version: 1.0.12
+version: 0.0.1
```

Comment:
Why is this commit needed and part of this PR?

I read the commit message `This changes the pubspec version to run a test build on TestFlight that will not be shown on the top but on version 0.0.1` but I don't understand *why* this was necessary?

Reply:
read the commit message 😜 : [Remove this commit before merging](https://github.com/get10101/10101/pull/534/commits/c4c2886f3c4fa538ab17e5f43b5b68ad89a1e200)